modules/nodes-scheduler-node-projects-configuring.adoc

```diff
@@ -5,148 +5,39 @@
 [id='nodes-scheduler-node-projects-configuring_{context}']
 = Configuring the Pod Node Selector admission controller to use projects in {product-title}
 
-// sjenning will need rewrite for 4.0
-
 You can configure the Pod Node Selector admission controller to ensure that pods are only placed onto nodes in specific projects.
+The Pod Node Selector admission controller uses a configuration file to set options for the behavior of the backend. 
 
 .Procedure
 
-To activate the *Pod Node Selector* admission controller: 
-
-. Configure the *Pod Node Selector* admission controller and whitelist, using one of the following methods: 
-
-** Add the following to the master configuration file, *_/etc/origin/master/master-config.yaml_*:
-+
-----
-admissionConfig:
-  pluginConfig:
-    PodNodeSelector:
-      configuration:
-        podNodeSelectorPluginConfig: <1>
-          clusterDefaultNodeSelector: "k3=v3" <2>
-          ns1: region=west,env=test,infra=fedora,os=fedora <3>
-----
-+
-<1> Adds the *Pod Node Selector* admission controller plug-in.
-<2> Creates default labels for all nodes.
-<3> Creates a whitelist of permitted labels in the specified project. Here, the project is `ns1` and the labels are the `key=value` pairs that follow.
-
-** Create a file containing the admission controller information:
+. Create a file containing the admission controller information:
 +
+[source,yaml]
 ----
 podNodeSelectorPluginConfig:
-    clusterDefaultNodeSelector: "k3=v3"
-     ns1: region=west,env=test,infra=fedora,os=fedora
+ clusterDefaultNodeSelector: name-of-node-selector
+ namespace1: name-of-node-selector
```

Comment:
is namespace1  the same as ns1 as below?
I not quite understand the example,  namespace1 and namespace2 have the same value

Reply:
@weinliu Thank yo for your comments. Your pointing out these issues led me to learn that the defaultNodeselector is not yet in 4.1 at this time and not clear if the feature will get in. I commented-out the topic in the _topic.yaml file, hiding it from users until the feature is in; at which time I will work in your comments. 
I believe you are correct that I have this procedure wrong. 
PR for commenting out the feature:
https://github.com/openshift/openshift-docs/pull/14590